__tests__/pages/kontak-darurat.test.tsx

```diff
@@ -0,0 +1,161 @@
+    expect(title).toBeVisible();
+
+    const breadcrumbs = screen.getByText(/^Kontak Darurat$/i);
+    expect(breadcrumbs).toBeVisible();
+    expect(breadcrumbs).toHaveAttribute("href", "/kontak-darurat");
+  });
+
+  it("renders the emergency contact sections correctly", () => {
+    render(<EmergencyContactSection emergency_contacts={emergency_contacts} />);
+    const description = screen.getByText(
+      /Kumpulan informasi situs dan kontak penting terkait fasilitas serta alat kesehatan untuk COVID-19 di level nasional./i,
+    );
+    expect(description).toBeVisible();
+    const faskes_title = screen.getByText(
+      /Cari kontak penyedia faskes secara detail perprovinsi se-Indonesia/i,
+    );
+    expect(faskes_title).toBeVisible();
+    const faskes_button = screen.getByText(/Telusuri Sekarang/i);
+    expect(faskes_button).toBeVisible();
+    const hotline_button = screen.getByText(/Hubungi 119 ext. 9/i);
+    expect(hotline_button).toBeVisible();
+    const hotline_title = screen.getByText(/Hotline Kementerian Kesehatan/i);
+    expect(hotline_title).toBeVisible();
+  });
+
+  it("renders the emergency contact card correctly", () => {
+    render(
+      <EmergencyContactCard
+        description={contact.description}
+        image={contact.image}
+        name={contact.name}
+        url={contact.url}
+      />,
+    );
+    const cardImage = screen.getByAltText(
+      `kontak darurat covid ${contact.name}`,
+    );
+    expect(cardImage).toBeVisible();
+    const contact_name = screen.getByText(contact.name);
+    expect(contact_name).toBeVisible();
+    const contact_description = screen.getByText(contact.description);
+    expect(contact_description).toBeVisible();
+    const contact_button = screen.getByTestId(`contact-button-${contact.name}`);
+    expect(contact_button).toBeVisible();
+    expect(contact_button).toHaveAttribute("href", contact.url);
+  });
+
+  it("renders the chatbot section correctly", () => {
+    render(<ChatbotSection />);
+    const chatbot_title = screen.getByText(/CovidAsha Chatbot 24x7/i);
+    expect(chatbot_title).toBeVisible();
+    const chatbot_description = screen.getByText(
+      /Layanan berbasis chatbot untuk mencari semua informasi terkait COVID-19/i,
+    );
+    expect(chatbot_description).toBeVisible();
+    const chatbot_button = screen.getByText(/Kirim chat sekarang/i);
+    expect(chatbot_button).toBeVisible();
+    expect(chatbot_button).toHaveAttribute(
+      "href",
+      "https://bit.ly/hotlinewarga",
+    );
+  });
+
+  it("renders the vaccine disclosure correctly", () => {
+    render(<VaccineSection vaccine_section={vaccine_section} />);
+    const disclosure_title = screen.getByText(/Mau Vaksin COVID-19?/i);
+    expect(disclosure_title).toBeVisible();
+
+    expect(screen.queryByText(vaccine.title)).toBeNull();
+    fireEvent.click(screen.getByTestId("chevron-down-icon"));
+    expect(screen.queryByText(vaccine.title)).toBeVisible();
```

Comment:
```suggestion
    expect(screen.queryByText(vaccine.title)).not.toBeInTheDocument();
    userEvent.click(screen.getByTestId("chevron-down-icon"));
    expect(screen.getByText(vaccine.title)).toBeVisible();
```

Please use the`.not.toBeInTheDocument()` assertion from `jest-dom` to make the error message more obvious when the test is failing.
Please use [userEvent instead of fireEvent](https://testing-library.com/docs/guide-events/#interactions-vs-events).
When you're querying for visibility, please use `getByText` instead, because it will throw an error if the element can't be found and it's what we're expecting from assertion of elements that's supposed to be visible.